build.gradle

```diff
@@ -28,6 +28,10 @@ ext {
 
 dependencies {
     testImplementation("junit:junit:4.12")
+    testImplementation("jmock:jmock:1.+")
+    testImplementation("org.testng:testng:6.14.2")
+    testImplementation('nl.javadude.assumeng:assumeng:1.2.4')
+    testImplementation('org.openmicroscopy:omero-common-test:5.5.0-m4')
```

Comment:
Will this get automatically updated by our scripts? (Doesn't block this PR.)

Reply:
I have not looked at that.

Reply:
Might just mean we need a process to remember to adjust when we ought.